controllers/feeds/commentChild.js

```diff
@@ -54,12 +54,13 @@ module.exports = async (req, res) => {
       data: {
         feed_id: req.body.activityId,
         type: 'reaction'
-      }
+      },
+      token: userToken.token
     };
     if (userToken) {
       if (detailUser.user_id !== detailSendUser.user_id) {
         messaging()
-          .sendToDevice(userToken.token, payload)
+          .send(payload)
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The check for `userToken` should be done before trying to access `userToken.token`. If `userToken` is `null` or `undefined`, this will throw an error. Also, the empty `.then(() => {})` can be removed as it does nothing.

```diff
-      token: userToken.token
-     };
-     if (userToken) {
+     };
+     if (userToken) {
+       payload.token = userToken.token;
       if (detailUser.user_id !== detailSendUser.user_id) {
         messaging()
          .send(payload)
-          .then(() => {});
       }
     }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->